test_runner/performance/test_branch_creation.py

```diff
@@ -140,10 +140,13 @@ def test_branch_creation_many(neon_compare: NeonCompare, n_branches: int, shape:
 
     # start without gc so we can time compaction with less noise; use shorter
     # period for compaction so it starts earlier
+    def patch_default_tenant_config(config):
+        config["compaction_period"] = "3s"
+        config["gc_period"] = "0s"
+        return True
+
+    env.pageserver.edit_config_toml(patch_default_tenant_config)
```

Comment:
this looks like it edits the main keys of pageserver config, not the tenant config anymore.

Reply:
66fc413a72685caef015d3a257b4694419d92945